toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -12,10 +12,14 @@ import (
 
 // SystemConfig defines how each system present on the image is supposed to be configured.
 type SystemConfig struct {
-	Hostname             string                    `yaml:"Hostname"`
-	AdditionalFiles      map[string]FileConfigList `yaml:"AdditionalFiles"`
-	PostInstallScripts   []Script                  `yaml:"PostInstallScripts"`
-	FinalizeImageScripts []Script                  `yaml:"FinalizeImageScripts"`
+	Hostname              string                    `yaml:"Hostname"`
+	PackagesToAddLists    []string                  `yaml:"PackagesToAddLists"`
+	PackagesToAdd         []string                  `yaml:"PackagesToAdd"`
```

Comment:
Is this meant to be `[]string`, or use the new `PackageList` type?

Reply:
PackageLists contains file paths to a separate YAML file that contains the `PackageList` type.

Reply:
Changed the names.